server/service/src/invoice/inbound_shipment/update/generate.rs

```diff
@@ -385,36 +388,34 @@ pub fn generate_lines_and_stock_lines(
             ..
         }: InvoiceLineRow = invoice_line;
 
-        if number_of_packs > 0.0 {
```

Comment:
Ah yea this was a minor side quest - we were skipping it in the end so why do all the work earlier in the loop?